presets/ragengine/models.py

```diff
@@ -122,6 +122,15 @@ def messages_to_prompt(messages: list[dict]) -> str:
     return "\n".join(string_messages)
 
 
+def messages_to_simplified_messages(messages: list[dict]):
```

Comment:
Can you add some examples in a comment to explain what exactly the `get_message_content` and `message_to_simplified_messages` methods do?